packages/repack/android/src/newarch/java/com/callstack/repack/ChunkManagerModule.kt

```diff
@@ -0,0 +1,25 @@
+package com.callstack.repack
+
+import android.os.Handler
+import com.facebook.react.bridge.*
+
+class ScriptManagerModule(reactContext: ReactApplicationContext) : NativeScriptManagerSpec(reactContext) {
+    private val remoteLoader: RemoteScriptLoader = RemoteScriptLoader(reactApplicationContext)
+    private val fileSystemLoader: FileSystemScriptLoader = FileSystemScriptLoader(reactApplicationContext)
+
+    private var implementation: ScriptManagerModuleImpl = ScriptManagerModuleImpl()
```

Comment:
Why do we need to initialize the shared implementation class? I think a better way to do this is just calling methods on it, same as pager-view does: https://github.com/callstack/react-native-pager-view/blob/16173bd3222305eebc5c77bf326aab448c712717/android/src/paper/java/com/reactnativepagerview/PagerViewViewManager.kt#L79

Reply:
Fixed.